lib/curl_req.ex

```diff
@@ -28,8 +28,24 @@ defmodule CurlReq do
     req
   end
 
-  defp run_steps(req) do
-    Enum.reduce(req.request_steps, req, fn {step_name, step}, req ->
+  defp run_steps(req, option) do
+    req.request_steps
+    |> Enum.filter(fn {step_name, _} ->
+      case option do
+        _ when is_boolean(option) ->
+          option
+
+        [{:except, excludes}] ->
```

Comment:
Can you add a test for this, so that it becomes clearer what the purpose is of this addition? (same applies for `:only`).